api/include/opentelemetry/version.h

```diff
@@ -5,8 +5,8 @@
 
 #include "opentelemetry/detail/preprocessor.h"
 
-#define OPENTELEMETRY_ABI_VERSION_NO 0
-#define OPENTELEMETRY_VERSION "1.0.0-rc4"
+#define OPENTELEMETRY_ABI_VERSION_NO 1
```

Comment:
This looks a breaking change, as it change our namespace. Should we document this separately?

Reply:
Have updated the CHANGELOG.md to reflect the change.